src/query/functions/temporal/rate.go

```diff
@@ -221,7 +223,8 @@ func irateFunc(
 	datapoints ts.Datapoints,
 	isRate bool,
 	_ bool,
-	timeSpec transform.TimeSpec,
+	_ int64,
+	_ int64,
```

Comment:
Can we use `xtime.UnixNano` here too? https://github.com/m3db/m3/blob/master/src/x/time/unix_nano.go#L27

Reply:
Good call, changed all of these in the upcoming PR